src/components/home.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
+import { useSelector, shallowEqual, useDispatch } from 'react-redux';
+import { fetchCountry } from '../redux/countries/fetchcountries';
+import AllCountry from './allCountries';
+import europe from '../images/europe.png';
+import '../App.css';
+
+let retriveData = true;
+const HomePage = () => {
```

Comment:
- [ ] As requested by the previous reviewer and as stipulated in the requirements of this project, the Home page should have a list of categories that could be filtered using a given parameter, e.g. country name. We are expected to filter data retrieved from the API using a **Filter** stateless component. I think you'll require a search field on the Home page like the image below to achieve this. Filtering should probably be done using the ES6 filter() method.
![Screenshot from 2022-12-07 14-16-21](https://user-images.githubusercontent.com/63087491/206189032-bba6f0cf-d64a-4d9c-bd75-f9c093d96db6.png)
NB. The image is just for demonstrative purposes.

- As you can see, a user needs to scroll through the entire page to select a country right below which is very tedious and does not give a good user experience while using your app. Let's be able to filter data using the search field.